solidity/ecdsa/contracts/WalletRegistry.sol

```diff
@@ -96,17 +101,26 @@ contract WalletRegistry is Ownable {
         IERC20 _tToken,
         IWalletStaking _staking,
         DKGValidator _dkgValidator,
+        IRandomBeacon _randomBeacon,
         address _walletOwner
     ) {
         sortitionPool = _sortitionPool;
         tToken = _tToken;
         staking = _staking;
+        randomBeacon = _randomBeacon;
```

Comment:
@dimpar @pdyraga Thoughts on having `randomBeacon` governable?

Reply:
I take that it might be useful in case of an update of `randomBeacon`?

Reply:
Yes, we need to be able to upgrade it.